scheduler/test/testdata.go

```diff
@@ -143,6 +145,46 @@ var PodForHandleTest = &v1.Pod{
 	Spec: v1.PodSpec{NodeName: NodeSmall.Name},
 }
 
+var PodForHandlePod = &v1.Pod{
+	ObjectMeta: apimeta.ObjectMeta{
+		Name:      "handle-pod",
+		Namespace: "test",
+		UID:       "384900cd-dc7b-41ec-837e-9c4c1762363e",
+	},
+	Spec: v1.PodSpec{NodeName: NodeSmall.Name},
+}
+
+// waitingPod implements the framework.WaitingPod interface
+type waitingPod struct {
+	pod            *v1.Pod
+	pendingPlugins map[string]*time.Timer
+	mu             sync.RWMutex
+}
+
+func (wp *waitingPod) GetPod() *v1.Pod {
+	return wp.pod
+}
+
+func (wp *waitingPod) GetPendingPlugins() []string {
+	wp.mu.RLock()
+	defer wp.mu.RUnlock()
+	var plugins []string
+	for plugin := range wp.pendingPlugins {
+		plugins = append(plugins, plugin)
+	}
+	return plugins
+}
+
+func (wp *waitingPod) Allow(pluginName string) {
+	wp.mu.Lock()
+	defer wp.mu.Unlock()
+}
+
+func (wp *waitingPod) Reject(reason string, msg string) {
+	wp.mu.Lock()
+	defer wp.mu.Unlock()
+}
```

Comment:
any reason you implemented it here, not at scheduler/test/util.go?

Reply:
i will move this into `scheduler/test/util.go`